src/DynamoCLI/Program.cs

```diff
@@ -92,6 +92,11 @@ private static DynamoModel StartupDynamo(StartupUtils.CommandLineArguments cmdLi
                 model = Dynamo.Applications.StartupUtils.MakeModel(true, string.Empty, cmdLineArgs.AnalyticsInfo);
             }
 
+            if (!string.IsNullOrEmpty(cmdLineArgs.CERLocation))
+            {
+                model.CERLocation = cmdLineArgs.CERLocation;
```

Comment:
so this will override whatever was passed in the startup config?

Reply:
Yes.